ci-operator/step-registry/mirror-images/by-oc-mirror/mirror-images-by-oc-mirror-commands.sh

```diff
@@ -29,6 +29,21 @@ function run_command() {
     eval "${CMD}"
 }
 
+function isPreVersion() {
+  local required_ocp_version="$1"
+  local isPre version
+  curl -sL https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64 > /tmp/jq
+  chmod ug+x /tmp/jq
+  version=$(${oc_mirror_bin} version --v2 --output json | /tmp/jq -r '.clientVersion.gitVersion' | tail -n 1 | cut -d '.' -f1,2)
```

Comment:
```suggestion
  version=$(${oc_mirror_bin} version --v2 --output json | python3 -c 'import json,sys;j=json.load(sys.stdin);print(j["clientVersion"]["gitVersion"])' | cut -d '.' -f1,2)
```

Downloading jq binary is not good idea, because there is some connection limitation from github, especially when multi downloading happening in the same time.

Reply:
thanks so much